client/jetpack-connect/connection-notice-types.js

```diff
@@ -22,6 +22,7 @@ export const NOT_EXISTS = 'notExists';
 export const NOT_JETPACK = 'notJetpack';
 export const NOT_WORDPRESS = 'notWordPress';
 export const OUTDATED_JETPACK = 'outdatedJetpack';
+export const INVALID_CREDENTIALS = 'invalidCredentials';
```

Comment:
The definition of `LOGIN_FAILURE` becomes obsolete, am I right?

Reply:
Oh well spotted. Seems like it should be `'loginFailure'` to match the others.